soc/xtensa/intel_adsp/tools/cavstool_client.py

```diff
@@ -41,27 +43,57 @@ def send_cmd(self, cmd):
 
             if ack == CMD_LOG_START:
                 self.monitor_log()
-            elif ack == CMD_LOG_STOP:
-                log.info(f"Stop output.")
             elif ack == CMD_DOWNLOAD:
                 self.run()
             else:
                 log.error(f"Receive incorrect msg:{ack} expect:{cmd}")
 
-    def download(self, filename):
+    def uploading(self, filename):
         # Send the FW to server
+        fname = os.path.basename(filename)
+        fsize = os.path.getsize(filename)
+        flen = len(fname)
```

Comment:
You are right, the filename length is not used here because we have a fixed filename. At first I just want to make 4 bytes align. Let's remove them and keep it clean. Thanks.